cookbook-2.md

```diff
@@ -10,100 +10,168 @@ As Service Consumer, call the API of a Service Provider while being pre-authoriz
 
```

Comment:
bovenstaande is incorrect / onduidelijk; de reden dat je moet authenticeren met een association registry hier is omdat je anders niet weet waar het authorisation registry is. je moet je ook nog apart authenticeren (ophalen access token) bij het authorisation registry en bij de uiteindelijke service provider